src/lib.rs

```diff
@@ -3584,11 +3595,11 @@ impl Build {
                 })
             })
             .copied()
-            // If no toolchain was found, provide the first toolchain that was passed in.
-            // This toolchain has been shown not to exist, however it will appear in the
-            // error that is shown to the user which should make it easier to search for
-            // where it should be obtained.
-            .or_else(|| prefixes.first().copied())
+        // If no toolchain was found, provide the first toolchain that was passed in.
+        // This toolchain has been shown not to exist, however it will appear in the
+        // error that is shown to the user which should make it easier to search for
+        // where it should be obtained.
+        .or_else(|| prefixes.first().copied())
```

Comment:
I think you need to revert this change as it breaks cargo-fmt pipeline

Reply:
Yeah, JetBrains formatter is kinda broken sometimes.